src/App.jsx

```diff
@@ -68,7 +69,7 @@ export default function App() {
     return (
         <main className="flex flex-col items-center justify-between h-full p-4 mt-16 ">
             {tenzies && <Confetti />}
-            <h1 className="m-auto text-4xl font-bold ">Tenzies</h1>
+            <h2 className="m-auto text-4xl font-bold ">Tenzies</h2>
             <p className="items-center p-4 mt-0 text-lg font-normal text-center text-gray-700">Roll until all dice are the same. 
             Click each die to freeze it at its current value between rolls.</p>
```

Comment:
As these elements are not directly depending to the functions and the states, move them into their own component to make your codes more clean.